CHANGELOG.md

```diff
@@ -12,12 +12,15 @@ a release.
 ```markdown
 ## [3.6.1] - 2022-07-26
 ### Fixed
-- Sortable: Fix issue with add+delete position synchronization (#1932)
+- SoftDeleteable: Fix issue with soft-deleting objects which proxy was not yet initialized
```

Comment:
Please, use the `[Unreleased]` section to describe the changes related to this PR.

Reply:
Sorry I mistakenly added the line to the wrong section. I've corrected it and updated the PR.